tests/integration/tier0/destructive/isolated-system-conversion/prepare_system.py

```diff
@@ -3,7 +3,7 @@
 
 import pytest
 
-from conftest import TEST_VARS, SystemInformationRelease
+from conftest import TEST_VARS, SystemInformationRelease, _get_full_kernel_title, grub_setup_workaround
```

Comment:
`_get_full_kernel_title` is a private function and should not be imported.

Reply:
Good catch, I was moving it around and forgot.